website/docs/r/is_vpc.html.markdown

```diff
@@ -54,6 +54,7 @@ Review the argument references that you can specify for your resource.
 - `default_security_group_name` - (Optional, String) Enter the name of the default security group.
 - `default_routing_table_name` - (Optional, String) Enter the name of the default routing table.
 - `name` - (Required, String) Enter a name for your VPC. No.
+- `no_sg_acl_rules` - (Optional, Bool) Delete all rules attached to default security group and default network ACL for a new VPC. This attribute has no impact on update.
```

Comment:
`If set to true, delete all rules attached to default security group and default network ACL for a new VPC. This attribute has no impact on update. default false`